src/backend/aspen/api/schemas/samples.py

```diff
@@ -1,12 +1,45 @@
+from typing import List, Optional
+
 from pydantic import constr
 
-from aspen.api.schemas.base import BaseRequest
+from aspen.api.schemas.base import BaseRequest, BaseResponse
 
 
 class SampleRequestSchema(BaseRequest):
     # mypy + pydantic is a work in progress: https://github.com/samuelcolvin/pydantic/issues/156
     name: constr(min_length=1, max_length=128, strict=True)  # type: ignore
 
 
-class SampleResponseSchema(BaseRequest):
-    name: str
+class SampleGisaidResponseSchema(BaseResponse):
+    gisaid_id: Optional[str]
+    status: str
+
+
+class SampleLineageResponseSchema(BaseResponse):
+    last_updated: Optional[str]
+    lineage: Optional[str]
+    probability: Optional[str]
+    version: Optional[str]
+
+
+class SampleGroupResponseSchema(BaseResponse):
+    group_id: int
+    group_name: str
+
+
+class SampleResponseSchema(BaseResponse):
+    collection_date: str
+    collection_location: str
+    czb_failed_genome_recovery: bool
+    gisaid: SampleGisaidResponseSchema
+    lineage: SampleLineageResponseSchema
+    private: bool
+    private_identifier: str
+    public_identifier: str
+    sequencing_date: str = "N/A"
```

Comment:
I agree with this, but right now when dates are missing the convention is that it comes in as the string `N/A`. This was previously handled by the `format_date` and `format_datetime` util funcs.

We probably should break with that -- I agree with Jess, the FE should decide how to display null data -- but switching over to having date be `null` or just missing entirely from response will require corresponding changes to how FE handles and parses. I think it's best to just follow the convention for now and file a ticket to come back and convert the response for missing dates later.

Reply:
(To be explicit, there is currently code in the FE that is literally doing ` blah === "N/A"`, I don't think there's good handling for null/missing on the other hand)

Reply:
Yeah, let's fix the frontend as well.

Reply:
Null responses are working fine in my frontend, they just default to `---`.

Reply:
They work, but -- pretty sure -- they aren't working 100%? There's a general default that causes the `---` to appear for any table cell that is missing data (actually, looking at code right now, it's false-y, which is a different problem, sometimes you want to display `0`, but that's a tangent that has nothing to do with the BE). So while it won't break, I think the sorting and/or filtering might get mucked up?

There's definitely stuff in the FE that is looking for `"N/A"` as the canonical no-data for date. It's bad! Don't get me wrong! But I think for now we should just leave the date serializing for the `null` case as `"N/A"`. I think it will take more work on the FE to fix than the BE, so my vote is to plan to fix the FE in the near future, and as part of that work change over the BE serializing for dates (since that should just be a line or two of code).

My work on frontend table refactoring will be touching that code, so I could probably handle it as part of my work then. I can spawn a new ticket to track this so it doesn't get dropped if I wind up missing it during the refactor. Let me know if you're fine with that and I'll write the ticket. If you'd rather handle it now, just make sure to verify that filtering and column sorting are working correctly as well. Also, we should potentially check with Product/Design how they want the no-data case displayed for dates: on the Sequencing Date FE column the design was for it to be simply `-` instead of triple dash. [You can check out `src/frontend/src/views/Data/cellRenderers.tsx`, Line 118 to see how I had to make that happen. It's a bit ugly, but it's a limitation of how our table rendering currently works. I'm planning to make this easier in my refactor.]

Reply:
We need to modify the frontend code to point to the new endpoint anyway -- we can update the FE code to handle null dates at the same time.